src/sbd-pacemaker.c

```diff
@@ -103,6 +103,9 @@ static pe_working_set_t *data_set = NULL;
 
 static long last_refresh = 0;
 
+static int pcmk_clean_shutdown = 0;
+static int pcmk_shutdown = 0;
```

Comment:
Not necessary to change, but bool would make the purpose more clear (or using 1=shutdown, 2=clean shutdown with either a comment to that effect or an enum)

Reply:
Was adapting to the surrounding code that as well uses ints for boolean content for the first.
For longer term I was rather thinking of a nice descriptive enum that might cover at least parts of the remaining state as well.

Reply:
Sounds good